src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -205,8 +208,12 @@ protected Object decode(
                                 case 3:
                                     position.setDeviceTime(new Date(buf.readUnsignedIntLE() * 1000));
                                     break;
+                                case 8:
+                                    value = buf.readUnsignedByte();
+                                    position.setValid(BitUtil.check(value, 1));
+                                    position.set(Position.KEY_SATELLITES, ((value & 0b11111100) >> 2));
```

Comment:
Probably the last thing. It looks like you haven't updated this to use `BitUtil`.

Reply:
Oh right, I'll fix it now.